core/shared/src/main/scala/matryoshka/patterns/Diffable.scala

```diff
@@ -25,7 +25,7 @@ import scalaz._, Scalaz._
 import simulacrum._
 
 @typeclass trait Diffable[F[_]] { self =>
-  private implicit def self0 = self
+  private implicit def selfʹ: Diffable[F] = self
```

Comment:
ah, I was searching for references to it but hadn't realized it was implicit. I mean that still sounds weird, but I imagine it makes sense